runners/metrics_exporter.go

```diff
@@ -79,8 +131,16 @@ func (m *metricsExporter) Start(ctx context.Context) error {
 			case <-ctx.Done():
 				return
 			case met := <-metricsCh:
-				m.availableBytes.WithLabelValues(met.DeviceClass).Set(float64(met.FreeBytes))
-				m.sizeBytes.WithLabelValues(met.DeviceClass).Set(float64(met.SizeBytes))
+				if met.DeviceClassType == TypeThick {
+					// metrics for volumegroup subsystem
+					m.availableBytes.WithLabelValues(met.DeviceClass).Set(float64(met.FreeBytes))
+					m.sizeBytes.WithLabelValues(met.DeviceClass).Set(float64(met.SizeBytes))
+				} else if met.DeviceClassType == TypeThin {
+					// metrics for thinpool subsystem
+					m.thinPool.tpSizeBytes.WithLabelValues(met.DeviceClass).Set(float64(met.SizeBytes))
```

Comment:
Can you also include the thin pool name in the labels?

Reply:
It looks better to use other metrics like `kube_persistentvolume_info` to describe VG/pool name and another property. If this feature is not an urgent requirement, Would you discuss it later?

Reply:
Not an urgent requirement.  We can discuss this later.